Classes/Setup.php

```diff
@@ -170,18 +170,27 @@ public static function schema(array $sql) : array
 
         foreach ($connectionNames as $connectionName) {
             $conn = $ctx->db($connectionName);
+            $dbtype = $conn->getRawObject()->getParams()['adapter'];
```

Comment:
If you use `$ctx->config('resource/' . $connectionName . '/adapter')`, you don't have to distinguish between `pgsql` and `pdo_pgsql` any more

Reply:
Fixed, thanks for this pointer.